examples/ws_subscription.rs

```diff
@@ -54,10 +54,10 @@ async fn main() -> anyhow::Result<()> {
 
 async fn run_server() -> anyhow::Result<SocketAddr> {
 	let mut server = WsServer::new("127.0.0.1:0").await?;
-	let mut subscription = server.register_subscription("subscribe_hello", "unsubscribe_hello").unwrap();
+	let subscription = server.register_subscription::<String>("subscribe_hello", "unsubscribe_hello").unwrap();
```

Comment:
```suggestion
	let subscription = server.register_subscription::<()>("subscribe_hello", "unsubscribe_hello").unwrap();
```

To make it clear that the `P` type parameter is the parameters sent to the subscription (and there are none here).